src/js/bootstrap-slider.js

```diff
@@ -973,14 +983,23 @@ const windowIsDefined = (typeof window === "object");
 			- Any method that is not part of the public interface.
 			- Place it underneath this comment block and write its signature like so:
 
-			  					_fnName : function() {...}
+				_fnName : function() {...}
 
 			********************************/
 			_removeSliderEventHandlers: function() {
 				// Remove keydown event listeners
 				this.handle1.removeEventListener("keydown", this.handle1Keydown, false);
 				this.handle2.removeEventListener("keydown", this.handle2Keydown, false);
 
+				//rmove the listeners from the ticks if they had their own listeners
+				if(this.options.ticks_tooltip){
+					var ticks = document.getElementsByClassName('slider-tick-container')[0].children;
+					for(var i = 0; i < ticks.length; i++ ){
+						ticks[i].removeEventListener('mouseenter', this.createTickMouseOverListener, false);
```

Comment:
this is what is currently happening:

```js
// This method is acting similar to your createTickMouseOverListener
function bindListener(element) {
    element.addEventListener("mouseenter", function() {
          console.log("yolo");
    });
}

// call your function
bindListener(myElementReference);

/*
   This will not actually remove the listener bound in the bindListener() function because the 
   function references for `bindListener()` and `function() { console.log("yolo"); }` are different
*/
myElementReference.removeEventListener("mouseenter", bindListener, false);

```